code/game/objects/items/tools/crowbar.dm

```diff
@@ -103,13 +103,13 @@
 	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, TRUE)
 	if(tool_behaviour == TOOL_CROWBAR)
 		tool_behaviour = TOOL_WIRECUTTER
-		to_chat(user, span_notice("You attach the cutting jaws to [src]."))
+		balloon_alert(user,	("[src] will now act as wirecutters."))
```

Comment:
Balloon alerts are meant to be very short and all lowercase with no punctuation.

```suggestion
		balloon_alert(user, "attached cutting jaws")
```